scripts/run-benchmarks-on-travis.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
Can we turn this bash script into a Julia script? A Julia script might be easier to work with for other team members. It’s helpful for other tasks described in #822.